Modules/Office365DSC/DSCResources/MSFT_O365AdminAuditLogConfig/MSFT_O365AdminAuditLogConfig.psm1

```diff
@@ -27,13 +27,13 @@ function Get-TargetResource
         throw "This resource cannot delete Managed Properties. Please make sure you set its Ensure value to Present."
```

Comment:
Just spotted this. Can you update the error message to make it relevant to your resource?

Reply:
Done